packages/frontend-2/lib/settings/graphql/queries.ts

```diff
@@ -12,3 +12,20 @@ export const settingsSidebarWorkspacesQuery = graphql(`
     }
   }
 `)
+
+export const settingsWorkspacesMembersQuery = graphql(`
+  query SettingsWorkspacesMembers($workspaceId: String!) {
+    workspace(id: $workspaceId) {
+      team {
+        role
+        id
+        user {
+          avatar
```

Comment:
[Always ask back for an `id`, wherever possible, to ensure that cache can be updated anywhere its needed: https://www.apollographql.com/docs/react/data/mutations/#include-modified-objects-in-mutation-responses](https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#2851514fbf9a406da31e430a57859565)